test/e2e/data/infrastructure-vsphere/kustomization/cluster-template.yaml

```diff
@@ -0,0 +1,211 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: '${ CLUSTER_NAME }'
+  name: '${ CLUSTER_NAME }'
+  namespace: '${ NAMESPACE }'
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
```

Comment:
@sedefsavas - this should be `192.168.30.0/24` ?

Reply:
Opened a PR for this: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1029